sql/changes/1.6/country_date_format.sql

```diff
@@ -0,0 +1,267 @@
+ALTER TABLE country
+  ADD date_format TEXT;

```

Comment:
I know Dojo must know from the start of the application which language will be used. We can tell it which language it should use or it can detect itself and load the requested locale data from the server.
If it detects the locale itself, it will also select the appropriate date setting. However, we don't allow Dojo to use its own detection: we provide our own override in the page header.